config/crd-sandbox/bases/integreatly.org_rhoamtenants.yaml

```diff
@@ -0,0 +1,55 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  creationTimestamp: null
+  name: rhoamtenants.integreatly.org
+spec:
+  group: integreatly.org
+  names:
+    kind: RhoamTenant
+    listKind: RhoamTenantList
+    plural: rhoamtenants
+    singular: rhoamtenant
+  scope: Cluster
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: RhoamTenant is the Schema for the RhoamTenants API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: RhoamTenantSpec defines the desired state of RhoamTenant
+            properties:
+              username:
```

Comment:
Hey Valery, does it make sense that we could remove the username field from the spec and just use the name of the CR? So, users would create a RHOAMTenant CR with a name equal to their user name?

Reply:
Done. Username removed, and done as suggested; working fine. Thank you Brian.